packages/ckeditor5-html-support/src/datafilter.js

```diff
@@ -208,17 +211,30 @@ export default class DataFilter extends Plugin {
 	}
 
 	/**
-	 * Matches and consumes allowed and disallowed view attributes and returns the allowed ones.
+	 * Processes all allowed and disallowed attributes on the view element by consuming them and returns allowed ones.
+	 *
+	 * This method applies the configuration set up by {@link #allowAttributes} and {@link #disallowAttributes}
```

Comment:
```suggestion
	 * This method applies the configuration set up by {@link #allowAttributes `allowAttributes()`} and {@link #disallowAttributes `disallowAttributes()`}
```

(may require reformatting due to max line length)